tests/integration.py

```diff
@@ -0,0 +1,9 @@
+from bw_interface_schemas import load_graph
+
+
+def test_dump_graph(bike_as_graph):
+    assert bike_as_graph.model_dump()
```

Comment:
suggestion (blocking): What about if we could test the result of the model_dump compared to the fixture ?